src/main/scala/com/gravity/goose/spark/GooseFetcher.scala

```diff
@@ -0,0 +1,123 @@
+package com.gravity.goose.spark
```

Comment:
Sorry for breaking this interface in previous release. I stay convinced that goose scraper must be brought as external deps to the project that we can explicitly ignore when not required. We'll keep it for now but may consider moving that elsewhere

Reply:
Good idea. We can work towards moving goose outside as external dependency soon.